data/core/tips.json

```diff
@@ -173,5 +173,120 @@
+    "category": "tip",
+    "text": [ "You can make unusual things from usual stuff.  Go full MacGyver style if needed." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "There are fungi you want to eat, and those you want to burn.  You'll know the difference." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Having a backup weapon is thoughtful.  As well as having a backup plan." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Windows can't stop zombies but they can stop gas, bile, and smoke.  And your smell." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Wet and angry?  A dry towel will bring your smile back." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Like to gamble?  Use mutagen!  Be a winner today!" ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Famous last words:  Lets poke this electiric zombie with a metal stick." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Smoking and napping is asking for more then a hole in your shirt." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "You're on fire?  Stop and wait to put it out." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Why it's called chicken walker?  Because you'll chicken out every time you'll see it." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Proceed with alertness and extreme prejudice.  Routine kills." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Most medicine doesn't work instantly, so just don't swallow whole bottle of pills." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Famous last words:  Lets poke this electric zombie with a metal stick." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "A full stomach will not solve your starvation.  Give it time and eat regulary." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Be mindful of enviromental protection of your gear.  It can take you places." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Don't overlook protective gear, like gas masks or turnout gear.  You never know." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Out of resources?  Breaking or disassembling stuff can yield intresting components." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Food's shelf life is not given.  Storage conditions matter, especially temperature." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Frozen food doesn't rot, but may be inedible if not deforsted." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Some dry  food can be eaten frozen, while other may become mushy after defrosting." ]
```

Comment:
Double space before "food".
Also better to write "some foods" and "while others".

Reply:
Also, "can be eaten frozen" and "may become mushy after defrosting" are not contrasting things. What's probably meant to be said here is:

`Though some kinds of dry food can be frozen and defrosted without an issue...` etc